src/concepts/pipeline/schemas.md

```diff
@@ -1,15 +1,14 @@
 # Generated Schemas
 
+<!-- toc -->
+
 ## Overview
 
 Schemas describe the structure of ingested data. They are used in the pipeline to validate the types
-and values of data, and to define a table schema in a data store. We use a repository of JSON Schemas
-to sort incoming data into [`decoded` and `error`
-datasets](../../cookbooks/bigquery/querying.md#projects-with-bigquery-datasets). We also generate BigQuery
-table schemas nightly from the JSON Schemas.
-
-This section is intended for those who want to modify the process of generating and applying schemas
-in various components of the data pipeline.
+and values of data, and to define a table schema in a data store. We use a repository of JSON
+Schemas to sort incoming data into [`decoded` and `error` datasets][bq-datasets]. We also generate
+BigQuery table schemas nightly from the JSON Schemas. You may also be also interested in the
+[`mozilla-pipeline-schemas` deploy dashboard][mps-deploys].
```

Comment:
let's be more explicit about why you might want to look at the deploy dashboard.

```suggestion
BigQuery table schemas nightly from the JSON Schemas: you can see the current status of this job on the [`mozilla-pipeline-schemas` deploy dashboard][mps-deploys].
```